drivers/serial/uart_mcux_lpuart.c

```diff
@@ -873,6 +865,7 @@ static int mcux_lpuart_rx_buf_rsp(const struct device *dev, uint8_t *buf, size_t
 	assert(data->async.next_rx_buffer_len == 0);
 	data->async.next_rx_buffer = buf;
 	data->async.next_rx_buffer_len = len;
+    uart_mcux_lpuart_dma_replace_rx_buffer(dev);
```

Comment:
you code indent is wrong should be 8

Reply:
thanks, I will reformat all the related indent and whitespaces.